src/Responder/FormattedResponder.php

```diff
@@ -49,8 +49,7 @@ public function __invoke(
         PayloadInterface $payload
     ) {
         if ($this->hasOutput($payload)) {
-            $formatter = $this->formatter($request);
-            $response = $this->format($response, $formatter, $payload);
```

Comment:
I'm not a fan of mixing this change in with adding exceptions. This PR should contain changes to exceptions **only** and other refactoring should be done in a separate PR.